api/open-positions.ts

```diff
@@ -0,0 +1,47 @@
+import Airtable from "airtable";
+import { NowRequest, NowResponse } from "@now/node";
+
+interface JobRecord {
+  "Stav": "Aktivní" | "Obsazeno";
+  "Úkol/Role": string;
+  "Odkaz na celou příležitost": string | null;
+  "Detaily": string | null;
+  "Časová náročnost": string | null;
+  "Urgentní poptávka": boolean;
+}
+
+interface Position {
+  title: string;
+  isOpen: boolean;
+  isUrgent: boolean;
+  detailUrl: string | null;
+  description: string | null;
+  timeRequirements: string | null;
+}
+
+export default async (req: NowRequest, res: NowResponse) => {
+  try {
+    const apiKey = process.env.AIRTABLE_API_KEY as string;
+    const table = new Airtable({ apiKey }).base("apppZX1QC3fl1RTBM")(
+      "Hledané role a úkoly - To Be Moved"
+    );
+    const records = await table.select({ view: "Otevřené role" }).all();
+    const out = JSON.stringify(records.map(parseRecord), null, 2);
+    res.setHeader("Content-Type", "application/json");
+    res.status(200).send(out);
```

Comment:
vůbec nevím, jak je na tom NowResponse, ale v Expressu lze použít `res.json()` a ušetřit si ruční nastavování hlavičky a JSON.stringify.

Reply:
_Vyřešeno ve mém PR._

Reply:
Já ten Typescript úplně strašlivě cargocultuju z různých předchozích skriptů a příkladů totiž 😁 Jde to, Honza už provedl ve svém PR (viz #4). Supr.

Reply:
Nikomu to neříkejte, ale dělám to samé. Pokud to pak mergnete, bude to vůbec můj první TS v životě.